### PR TITLE
DA-700 Mm/add biobank history

### PR DESCRIPTION
**Pull request by Seeker1911**
add history table and backfill.

### CODE REVIEW

File: rest-api/alembic/versions/e44e303ae759_add_biobank_version.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biobank_order', sa.Column('version', sa.Integer(), nullable=False))
```
[the remark this answers is not in the document]
Comment (danqrodney) [reply rest-api/alembic/versions/e44e303ae759_add_biobank_version.py]: I don't think you should need a schema migration in the other PR anymore?

File: rest-api/model/biobank_order.py [hunk 2]
```diff
@@ -116,3 +157,52 @@ class BiobankOrderedSample(Base):
...
   processed = Column('processed', UTCDateTime)
   finalized = Column('finalized', UTCDateTime)
+
+
+class BiobankOrderHistory(BiobankOrderBase, Base):
+  __tablename__ = 'biobank_history'
+
+  version = Column('version', Integer, primary_key=True)
+
+  @declared_attr
```
Comment (danqrodney) [inline rest-api/model/biobank_order.py]: You shouldn't need all of these here. They are already inherited from BiobankOrderBase.

File: rest-api/alembic/versions/e44e303ae759_add_biobank_version.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Seeker1911) [reply rest-api/alembic/versions/e44e303ae759_add_biobank_version.py]: will remove from the other PR 👍

File: rest-api/model/biobank_order.py (hunk 2 above)
Comment (Seeker1911) [reply rest-api/model/biobank_order.py]: oops. Think I yanked and pasted instead of delete and pasted. Removed.

**Pull request merged**